src/cloud/lib/i18n/ja.ts

```diff
@@ -502,6 +504,16 @@ const jpTranslation: TranslationSource = {
 
   [lngKeys.GeneralYes]: 'はい',
   [lngKeys.GeneralNo]: 'いいえ',
+
+  [lngKeys.ModalUnlockDashboardsTitle]: 'Unlock Dashboards',
+  [lngKeys.ModalUnlockDashboardsDescription]:
+    'Create unlimited dashboards to organize your docs, tasks.. in order to guide your workflow in your own way! Display the information the way you want to see it.',
```

Comment:
`your docs, tasks.. in order to ...` -> `your docs, tasks, etc., in order to ...`